mk/config.mk

```diff
@@ -626,3 +626,6 @@ CFG_COMPAT_GP10_DES ?= y
 
 # Defines a limit for many levels TAs may call each others.
 CFG_CORE_MAX_SYSCALL_RECURSION ?= 4
+
+# Compiles bget_main_test() to be called from a test TA
+CFG_TA_BGET_TEST ?= y
```

Comment:
If not setting test flags to "N" by default, then I think we should move flags like this under some generic flag
ifeq CFG_ENABLE_TEST ....
   CFG_TA_BGET_TEST ?= y
   CFG_TA_SOMETHING_ELSE_TEST ?= y
endif

The use case is when this is going to be used on end user / consumer devices. We want to avoid having test code enabled by default.

Reply:
Makes sense. This is an internal function that will get discarded if the function isn't used by the TA. The already present CFG_TEE_CORE_EMBED_INTERNAL_TESTS is a larger problem since that's in OP-TEE Core itself.
The best would be to do as you suggest. I'll add a commit fixing this.